### PR TITLE
docs PDF: add release-8.2 and remove archived versions

### PR DESCRIPTION
**Pull request by qiancai**
### **PR Type**
configuration changes

___

### **Description**
- Updated branch patterns in `docs-cn-postsubmits.yaml` and `docs-postsubmits.yaml` to include support for `release-8.2`.
- Removed support for `release-5.0` and `release-6.6` in both files.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>docs-cn-postsubmits.yaml</strong><dd><code>Update branch patterns for docs-cn postsubmits</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml

<li>Updated branch patterns for postsubmits.<br> <li> Added support for <code>release-8.2</code>.<br> <li> Removed support for <code>release-5.0</code> and <code>release-6.6</code>.<br>

</details>
    

  </td>
  <td><a href="https://github.com/PingCAP-QE/ci/pull/3020/files#diff-a8bd1b126b4211a56553a1a30b5216df53edaba89d5a577790b6832288b489e4">+4/-4</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>docs-postsubmits.yaml</strong><dd><code>Update branch patterns for docs postsubmits</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
prow-jobs/pingcap/docs/docs-postsubmits.yaml

<li>Updated branch patterns for postsubmits.<br> <li> Added support for <code>release-8.2</code>.<br> <li> Removed support for <code>release-5.0</code> and <code>release-6.6</code>.<br>

</details>
    

  </td>
  <td><a href="https://github.com/PingCAP-QE/ci/pull/3020/files#diff-b1066d5a5f50efe588f78cadcc9c9441d11c1b017c00a85627e2af77af140bf7">+4/-4</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ postsubmits:
       branches:
         - ^master$
         - ^release-[3-4].0$
```
Comment (qiancai) [inline prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml:12]:
```suggestion
```

File: prow-jobs/pingcap/docs/docs-postsubmits.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ postsubmits:
       branches:
         - ^master$
         - ^release-[3-4].0$
```
Comment (qiancai) [inline prow-jobs/pingcap/docs/docs-postsubmits.yaml:12]:
```suggestion
```

File: prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml [hunk 3]
```diff
@@ -10,7 +10,7 @@ postsubmits:
...
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-2]$
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml:16]:
```suggestion
        - ^release-8\.[0-9]+$
```

File: prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml (hunk 1 above)
Comment (wuhuizuo) [reply prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml:12]: LGTM

File: prow-jobs/pingcap/docs/docs-postsubmits.yaml [hunk 4]
```diff
@@ -9,8 +9,7 @@ postsubmits:
...
         - ^master$
-        - ^release-[3-4].0$
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-2]$
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/docs/docs-postsubmits.yaml:15]:
```suggestion
        - ^release-8\.[0-9]+$
```

**Review by wuhuizuo — APPROVED** [review]
/lgtm
/ok-to-test
/approve

File: prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml [hunk 5]
```diff
@@ -9,8 +9,7 @@ postsubmits:
...
         - ^master$
-        - ^release-[3-4].0$
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-9]+$
```
Comment (qiancai) [inline prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml:15]:
```suggestion
        - ^release-8\.[0-6]+$
```

File: prow-jobs/pingcap/docs/docs-postsubmits.yaml [hunk 6]
```diff
@@ -9,8 +9,7 @@ postsubmits:
...
         - ^master$
-        - ^release-[3-4].0$
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-9]+$
```
Comment (qiancai) [inline prow-jobs/pingcap/docs/docs-postsubmits.yaml:15]:
```suggestion
        - ^release-8\.[0-6]+$
```

File: prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml [hunk 7]
```diff
@@ -9,8 +9,7 @@ postsubmits:
...
         - ^master$
-        - ^release-[3-4].0$
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-6]+$
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/docs/docs-cn-postsubmits.yaml:15]:
```suggestion
        - ^release-8\.[0-6]$
```

File: prow-jobs/pingcap/docs/docs-postsubmits.yaml [hunk 8]
```diff
@@ -9,8 +9,7 @@ postsubmits:
...
         - ^master$
-        - ^release-[3-4].0$
-        - ^release-5\.[0-4]$
-        - ^release-6\.[156]$
-        - ^release-7\.[0-5]$
-        - ^release-8\.[0-1]$
+        - ^release-5\.[1-4]$
+        - ^release-6\.[15]$
+        - ^release-7\.[156]$
+        - ^release-8\.[0-6]+$
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/docs/docs-postsubmits.yaml:15]:
```suggestion
        - ^release-8\.[0-6]$
```

**Pull request merged**